src/phovea.ts

```diff
@@ -55,18 +55,32 @@ export default function (registry: IRegistry) {
     matches: '(targidSetSelection)'
   });
 
-  registry.push('ordinoStartMenuSection', 'targid_temporary_session', () => import('./menu/internal/TemporarySessionSection'), {
-    factory: 'new TemporarySessionSection',
+  registry.push('ordinoStartMenuSection', 'targid_current_session', () => import('./internal/components/session/CurrentSessionCard'), {
```

Comment:
I saw that you just repurposed the existing extension point `ordinoStartMenuSection`. However, that does not work anymore with the new start menu. We need new extension points for each tab.

Furthermore, import and use the EP constants (since we switched to phovea.ts file).